Source/WebGPU/WebGPU/ComputePipeline.mm

```diff
@@ -175,6 +178,12 @@ static MTLSize metalSize(auto workgroupSize, const HashMap<String, WGSL::Constan
     return m_pipelineLayout;
 }
 
+const BufferBindingSizesForBindGroup* ComputePipeline::bufferSizes(uint32_t index) const
+{
+    auto it = m_bufferSizes.find(index);
+    return it == m_bufferSizes.end() ? nullptr : &it->value;
```

Comment:
just to make sure I understand it: this might be null when the shader doesn't use any bindings in this bind group?

Reply:
or if there is a mismatch between the number of bind groups in the pipeline and the number used in the render pass. That may fail earlier due to existing validation, but did not want to rely on that